docs/swiftui/index.mdx

```diff
@@ -0,0 +1,38 @@
+---
+locale: en
+slug: /swiftui
+contentful_id: 7054sYGit32xDXHEHgjXPp
+icon: /img/light/swiftui.svg
+title: SwiftUI accessibility
+hide_table_of_contents: true
+description: SwiftUI has built-in accessibility features, accessibility APIs, and developer tools which provide an extraordinary opportunity to deliver a superior mobile experience to every user, including those with disabilities.
```

Comment:
Change description to: 

SwiftUI automatically provides accessibility support for common elements.  Developers can use SwiftUI's view modifiers to enhance and customize accessibility.